js/battle.js

```diff
@@ -719,16 +719,13 @@ var Pokemon = (function () {
 	Pokemon.prototype.hpWidth = function (maxWidth) {
 		if (this.fainted) return 0;
```

Comment:
I think it makes sense to add `|| !this.hp` here, and then add back the `|| 1` down there.